builds/misc/templates/base-image-publish/base-image-config.yaml

```diff
@@ -0,0 +1,17 @@
+steps:
+- pwsh: |
+    $info = New-Object System.Collections.ArrayList
+    # Tuple format:  (ModuleRepoPath,                           BaseImageName,                   ReturningVstsVar)
+    $info.AddRange((
+      [Tuple]::Create("edge-agent",                             "azureiotedge-agent-base",       "edgeAgentTag"),
+      [Tuple]::Create("edge-hub",                               "azureiotedge-hub-base",         "edgeHubTag"),
+      [Tuple]::Create("edge-modules/SimulatedTemperatureSensor","azureiotedge-module-base",      "moduleTag"),
+      [Tuple]::Create("test/modules/TestAnalyzer",              "azureiotedge-module-base-full", "fullModuleTag")
```

Comment:
shouldn't the API proxy be in that list?

Reply:
API proxy uses `arm32v7/haproxy:1.8.23` for ARM32 and `haproxy:1.8.23-alpine ` for AMD64. 
I don't think they are relevant to ARM base images `3.1.10-bionic-arm*` that we are updating here.

With that said, I have a few follow up question: 
1. How do we known when we should upgrade the base image for HAProxy e.g. OS security patch, bug fixes?
2. Are the HAProxy base image somehow related to the Bionic arm image releases? Are they release about the same frequency?